ddtrace/tracer/span_test.go

```diff
@@ -214,7 +214,7 @@ func TestShouldDrop(t *testing.T) {
 	} {
 		t.Run("", func(t *testing.T) {
 			s := newSpan("", "", "", 1, 1, 0)
-			s.SetTag(ext.SamplingPriority, tt.prio)
+			s.setSamplingPriority(tt.prio, samplernames.Default)
```

Comment:
quick question, why default here?

Reply:
afaik, the value we pass into `samplernames` doesn't make an impact for this test case, so I opted to keep the code clean (i.e. not adding if statements or extraneous parameters to the `tt` structs), since we would otherwise have to check for priorities `2` or `-1` to set it to `Manual`. 

`Default` says that the priority was set without a `sampler`, which I believe still holds true in this case. Maybe I'm missing some logic here?